index.html

```diff
@@ -1,47 +1,211 @@
+          sign-ups required.
+        </p>
+        <ul>
+          <li><button type="reset">html</button></li>
+          <li><button type="reset">css</button></li>
+          <li><button type="reset">javascript</button></li>
+        </ul>
+        <button type="submit">See Project></button>
+      </div>
+      <div class="modal">
+        <img src="./images/SnapshootPortfolio4.png" alt="" />
+        <h1 class="headtext">Multi-Post Stories</h1>
+        <ul>
+          <li class="canopy">CANOPY</li>
+          <li class="extra"><img src="./images/Counter.png" alt="" /></li>
+          <li class="extra">Back End Dev</li>
+          <li class="extra"><img src="./images/Counter.png" alt="" /></li>
+          <li class="extra">2015</li>
+        </ul>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul>
+          <li><button type="reset">html</button></li>
+          <li><button type="reset">css</button></li>
+          <li><button type="reset">javascript</button></li>
+        </ul>
+        <button type="submit">See Project></button>
+      </div>
+    </section>
+    <section class="about font align">
+      <h2 class="intro font">About Myself</h2>
+      <p>
+        Hello I'm a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <p class="connect font">LET'S CONNECT</p>
+      <ul class="social-media-icon">
+        <li>
+          <a href="#" class="Linkdin"
+            ><img src="./images/linkedIn.png" alt="Linkedin"
+          /></a>
+        </li>
+        <li>
+          <a href="#" class="twitter"
+            ><img src="./images/twitter.png" alt="Twitter"
+          /></a>
+        </li>
+        <li>
+          <a href="#" class="M"> <img src="./images/M.png" alt="M" /></a>
+        </li>
+        <li>
+          <a href="#" class="cat"><img src="./images/cat.png" alt="Cat" /></a>
+        </li>
+        <li>
+          <a href="#" class="Bingo"
+            ><img src="./images/bingo.png" alt="Bingo"
+          /></a>
+        </li>
+      </ul>
+      <button class="submit">Get my resume</button>
+      <ul class="skills">
+        <li>
+          <h2>Languages</h2>
+          <ul class="tray">
+            <li>
+              <img
+                class="piling"
+                src="./images/javascriptlogo.png"
+                alt=""
+              />Javascript
+            </li>
+            <li>
+              <img class="piling" src="./images/htmllogo.png" alt="" />Html
+            </li>
+            <li><img class="piling" src="./images/csslogo.png" alt="" />Css</li>
+          </ul>
+        </li>
+        <li><h2>Frameworks</h2></li>
+        <li><h2>Skills</h2></li>
+      </ul>
+      <hr />
+    </section>
```

Comment:
- [ ] As I stated with the other requirement it's very important that is at its best. Let's follow the design template to make our work look awesome. 👍🏾 

<html>
<body>
<!--StartFragment--><p dir="auto" style="box-sizing: border-box; margin-top: 0px !important; margin-bottom: 16px; color: rgb(201, 209, 217); font-family: -apple-system, BlinkMacSystemFont, &quot;Segoe UI&quot;, Helvetica, Arial, sans-serif, &quot;Apple Color Emoji&quot;, &quot;Segoe UI Emoji&quot;; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(13, 17, 23); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">Please ignore the green and orange borders around the elements</p>

Figma design | Your design
-- | --
![image](https://user-images.githubusercontent.com/29071984/170839572-5b057857-b3ba-4b6d-b57c-042f2d3a2183.png) |  ![image](https://user-images.githubusercontent.com/29071984/170839101-0f09db58-8abf-41c5-8d6f-d4be38dfa284.png)


<!--EndFragment-->
</body>
</html>